Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1041,7 +1041,10 @@ def update_remote_system_command(client, args):
 
             if not close_xdr_incident and (update_args.get('status') in XSOAR_RESOLVED_STATUS_TO_XDR.values()):
                 status = update_args.pop('status')
-                demisto.debug(f"Popped {status=} from update_args, incident status won't be updated in XDR.")
+                resolve_comment = update_args.pop('resolve_comment', None)
```

Comment:
Are you sure that the only field causing this issue is the `resolve_comment`?

Reply:
What about this?

Reply:
Yes, I checked other fields as closeNotes and it didnt make an error.